.travis.yml

```
@@ -8,3 +8,7 @@ jobs:
   allow_failures:
     - rust: nightly
   fast_finish: true
+script:
+  - cargo build --verbose --all
+  - cargo test --verbose --all
+  - cargo fmt -- --check
```

I think it's better to place `fmt` first - to make it fail fast